### PR TITLE
Feature/iat 3569

### PR DESCRIPTION
**Pull request by millerts**
This provides the list of resource summary objects for the dashboard.

### CODE REVIEW

File: tools-for-teachers-service/src/main/resources/application.sql.yml [hunk 1]
```diff
@@ -22,4 +22,10 @@ tft:
...
   deleteResourceLock: >-
-    DELETE FROM tft_resource_lock WHERE resource_id = :resourceId;
\ No newline at end of file
+    DELETE FROM tft_resource_lock WHERE resource_id = :resourceId;
+
+  findAllActiveResources: >-
+    SELECT tft.resource_id, tft.resource_type, tft.resource_json -> 'metadata' ->> 'title' AS "title", tft.resource_json -> 'metadata' -> 'administration' ->> 'status' AS "status", tft.inserted_by, tft.inserted_date FROM tft_resource tft
+    JOIN (select resource_id, MAX(d.inserted_date) AS "max_date" FROM tft_resource d WHERE d.action in(1,4) GROUP BY d.resource_id) AS mdj
+    ON mdj.resource_id = tft.resource_id AND mdj.max_date = tft.inserted_date
+    WHERE tft.action IN (1,4) ORDER BY inserted_date DESC;
```
Comment (malaffoon) [inline tools-for-teachers-service/src/main/resources/application.sql.yml]: I'm assuming that you invoked Alla to look at this. I know there won't be gazillions of records but this seems worth investigating performance? Maybe? Some day?

File: tools-for-teachers-service/src/main/java/org/opentestsystem/ap/tft/repository/ResourceRepository.java [hunk 2]
```diff
@@ -40,4 +43,11 @@
      * @return the persisted {@link ResourceEntity}
      */
     ResourceEntity save(ResourceEntity resourceEntity);
+
+    /**
+     * Returns all published or being created {@link ResourceSummary}
+     *
+     * @return the list of {@link ResourceSummary}
+     */
+    List<ResourceSummary> findAllActiveResources();
```
Comment (malaffoon) [inline tools-for-teachers-service/src/main/java/org/opentestsystem/ap/tft/repository/ResourceRepository.java]: What is the order of the list?

File: tools-for-teachers-service/src/main/resources/application.sql.yml (hunk 1 above)
Comment (millerts) [reply tools-for-teachers-service/src/main/resources/application.sql.yml]: I did not.  I can after the holidays and prior to deployment.  This is performant enough for UI development to move forward.

File: tools-for-teachers-service/src/main/java/org/opentestsystem/ap/tft/repository/ResourceRepository.java (hunk 2 above)
Comment (millerts) [reply tools-for-teachers-service/src/main/java/org/opentestsystem/ap/tft/repository/ResourceRepository.java]: Will fix

File: tools-for-teachers-service/src/main/java/org/opentestsystem/ap/tft/repository/ResourceRepository.java [hunk 3]
```diff
@@ -45,7 +45,7 @@
     ResourceEntity save(ResourceEntity resourceEntity);
 
     /**
-     * Returns all published or being created {@link ResourceSummary}
+     * Returns all published or being created {@link ResourceSummary} ordered by inseted date
```
Comment (malaffoon) [inline tools-for-teachers-service/src/main/java/org/opentestsystem/ap/tft/repository/ResourceRepository.java]: nit: typo

Comment (millerts) [reply tools-for-teachers-service/src/main/java/org/opentestsystem/ap/tft/repository/ResourceRepository.java]: Sigh.  Will fix.

**Pull request merged**